src/test/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionTest.java

```diff
@@ -124,121 +124,137 @@ private void runInOrderTest(Vp8PacketGenerator generator, int targetIndex)
     }
 
     /** Run an out-of-order test on a single stream, randomized order except for the first packet. */
-    private void runOutOfOrderTest(Vp8PacketGenerator generator, int targetIndex)
+    private void doRunOutOfOrderTest(Vp8PacketGenerator generator, int targetIndex,
+        long seed)
         throws RewriteException
     {
-        long seed = System.currentTimeMillis(); // Pass an explicit seed for reproducible tests
-        //  long seed = 1576267371838L;
+        DiagnosticContext diagnosticContext = new DiagnosticContext();
+        diagnosticContext.put("test", Thread.currentThread().getStackTrace()[2].getMethodName());
 
-        try
-        {
-            DiagnosticContext diagnosticContext = new DiagnosticContext();
-            diagnosticContext.put("test", Thread.currentThread().getStackTrace()[2].getMethodName());
+        RtpState initialState =
+            new RtpState(1, 10000, 1000000);
 
-            RtpState initialState =
-                new RtpState(1, 10000, 1000000);
+        final long expectedInitialTs = RtpUtils.applyTimestampDelta(initialState.maxTimestamp, 3000);
+        final long expectedTsOffset = RtpUtils.getTimestampDiff(expectedInitialTs, generator.ts);
 
-            final long expectedInitialTs = RtpUtils.applyTimestampDelta(initialState.maxTimestamp, 3000);
-            final long expectedTsOffset = RtpUtils.getTimestampDiff(expectedInitialTs, generator.ts);
+        final int reorderSize = 64;
+        ArrayList<PacketInfo> buffer = new ArrayList<>(reorderSize);
 
-            final int reorderSize = 64;
-            ArrayList<PacketInfo> buffer = new ArrayList<>(reorderSize);
+        for (int i = 0; i < reorderSize; i++)
+        {
+            buffer.add(generator.nextPacket());
+        }
 
-            for (int i = 0; i < reorderSize; i++)
-            {
-                buffer.add(generator.nextPacket());
-            }
+        Random random = new Random(seed);
```

Comment:
kotlintest actually has some support for this (random but including some pre-defined constants), maybe that'd work well here? https://github.com/kotlintest/kotlintest/blob/master/doc/reference.md#custom-generators

Reply:
Well, this is a Java test case...

Reply:
Oh, oops.